src/content/docs/tutorial-customer-experience/establish-current-state.mdx

```diff
@@ -0,0 +1,168 @@
+---
+title: "Establish your current state" 
+metaDescription: "New Relic observability maturity series: this guide gives you a quality foundation for optimizing your web application's performance to improve your customer experience."
+redirects:
+---
+
+import cxSegmentWhitelistInvestigation from 'images/cx_screenshot-full_segment-whitelist-investigation.webp'
+
+import cxQfDashboard from 'images/cx_screenshot-full_qf-dashboard.webp'
+
+import cxQfKpiProgress from 'images/cx_diagram_qf_kpi_progress.webp'
+
+The first step in any improvement experience is to establish your baselines so that you have numbers to measure your KPIs against and can track improvement over time, and improving your customer experience is no exception to this rule. Once you check to make sure everything you need is reporting correctly to our platform, you can validate your information and use our quality foundation dashboard to measure your customer experience.
```

Comment:
Intro paragraphs are pretty key to helping someone get into a state of reading a thing or deciding to. 

Can we get a little more casual?

"You want to improve your digital experience. How do you know if you've improved it or not? First, you need to know your baseline performance, so you have something to compare it to. Once you know how your experience should be performing, use New Relic [or specific named capabilities] to monitor that performance continually."

Or something like that. I think the key is to find the key point you want to make and focus on that, stripping out as much jargon as you can. How little can you write to get your main point across?